diff --git a/css-images/Overview.bs b/css-images/Overview.bs
index 5b658a8..66bd35b 100644
--- a/css-images/Overview.bs
+++ b/css-images/Overview.bs
@@ -38,6 +38,38 @@ Introduction {#intro}
 	replaced elements such as images within the box determined by the CSS layout algorithms.
 	It also defines in a generic way CSS's <a section href="#sizing">sizing algorithm</a> for images and other replaced elements.
 
+Module Interactions {#placement}
+--------------------------------
+
+	This module defines and extends the <<image>> value type
+	defined in [[!CSS-VALUES-3]].
+	It also replaces the <<url>> type with <<image>>
+	in the 'background-image', 'cursor', and 'list-style-image' definitions in CSS1 and CSS2
+	and adds <<image>> as an alternative to <<url>>
+	in the 'content' property's value.
+	It is presumed that CSS specifications beyond CSS2.1
+	will use the <<image>> notation in place of <<url>>
+	where 2D images are expected.
+	(See e.g. [[CSS-BACKGROUND-3]].)
+
+	None of the properties defined in this module, only 'image-rendering'
+	applies to <code>::first-line</code> and <code>::first-letter</code>.
+
+Values {#values}
+----------------
+
+	This specification follows
+	the <a href="https://www.w3.org/TR/CSS21/about.html#property-defs">CSS property definition conventions</a> from [[!CSS2]].
+	Value types not defined in this specification are defined in CSS Level 2 Revision 1 [[!CSS2]].
+	Other CSS modules may expand the definitions of these value types:
+	for example [[CSS-VALUES-3]], when combined with this module,
+	adds the ''initial'' keyword as a possible property value.
+
+	In addition to the property-specific values listed in their definitions,
+	all properties defined in this specification also accept the
+	<a href="https://www.w3.org/TR/CSS21/cascade.html#value-def-inherit">inherit</a>
+	keyword as their property value.
+	For readability it has not been repeated explicitly.
 
 <!--
    ██ ████ ██     ██    ███     ██████   ████████ ██
@@ -53,6 +85,10 @@ Image Values: the <<image>> type {#image-values}
 ================================================
 
 	The <<image>> value type denotes a 2D image.
+	It can be a
+	<a href="#url-notation">url reference</a>,
+	<a href="#cross-fade">cross-fade() function</a>,
+	or <a href="#gradients">gradient notation</a>.
 	Its syntax is:
 
 	<pre class="prod"><dfn>&lt;image></dfn> = <<url>> | <<image-set()>> | <<cross-fade()>> | <<gradient>></pre>
@@ -64,6 +100,11 @@ Image Values: the <<image>> type {#image-values}
 	In some cases, an image is invalid,
 	such as a <<url>> pointing to a resource that is not a valid image format.
 	An <dfn export lt="invalid image|valid image">invalid image</dfn> is rendered as a solid-color ''transparent'' image with no intrinsic dimensions.
+	However, <a>invalid images</a> can trigger error-handling clauses
+	in some contexts.
+	For example, an <a>invalid image</a> in 'list-style-image'
+	is treated as ''list-style-type/none'',
+	allowing the 'list-style-type' to render in its place. [[CSS21]]
 
 
 <!--
@@ -291,7 +332,7 @@ Gradients {#gradients}
 	Gradients are specified by defining the <dfn>starting point</dfn> and <dfn>ending point</dfn>
 	of a <dfn export>gradient line</dfn>
 	(which, depending on the type of gradient,
-	may be technically a line, or a ray, or a spiral),
+	may geometrically be a line, or a ray, or a spiral),
 	and then specifying colors at points along this line.
 	The colors are smoothly blended to fill in the rest of the line,
 	and then each type of gradient defines how to use the color of the <a>gradient line</a> to produce the actual gradient.
@@ -321,7 +362,7 @@ Linear Gradients: the ''linear-gradient()'' notation {#linear-gradients}
 
 <h4 class='no-toc' id='linear-gradient-syntax'>linear-gradient() syntax</h4>
 
-The linear gradient syntax is:
+	The linear gradient syntax is:
 
 	<pre class=prod>
 		<dfn>linear-gradient()</dfn> = linear-gradient(
@@ -409,9 +450,9 @@ The linear gradient syntax is:
 	</div>
 
 	The gradient's color stops are typically placed between the starting point and ending point on the <a>gradient line</a>,
-	but this isn't required -
+	but this isn't required:
 	the <a>gradient line</a> extends infinitely in both directions.
-	The starting point and ending point are merely arbitrary location markers -
+	The starting point and ending point are merely arbitrary location markers--
 	the starting point defines where 0%, 0px, etc are located when specifying color-stops,
 	and the ending point defines where 100% is located.
 	Color-stops are allowed to have positions before 0% or after 100%.
@@ -536,7 +577,7 @@ Radial Gradients: the ''radial-gradient()'' notation {#radial-gradients}
 	<dl dfn-type=value dfn-for="radial-gradient(), repeating-radial-gradient()">
 		<dt><dfn><<position>></dfn>
 		<dd>
-			Determines the <dfn dfn>gradient center</dfn> of the gradient.
+			Determines the <dfn dfn lt="gradient center">center</dfn> of the gradient.
 			The <<position>> value type
 			(which is also used for 'background-position')
 			is defined in [[!css-values-3]],
@@ -794,9 +835,6 @@ Repeating Gradients: the ''repeating-linear-gradient()'' and ''repeating-radial-
 
 	</div>
 
-	If the gradient has only a single color-stop,
-	it must render as a solid-color image equal to the color of that color-stop.
-
 	If the distance between the first and last color-stops is non-zero,
 	but is small enough that the implementation knows that the physical resolution of the output device is insufficient to faithfully render the gradient,
 	the implementation must <a lt="gradient-average-color">find the average color of the gradient</a>
diff --git a/css-values-3/Overview.bs b/css-values-3/Overview.bs
index e7c28ce..9ba2f83 100644
--- a/css-values-3/Overview.bs
+++ b/css-values-3/Overview.bs
@@ -1375,7 +1375,10 @@ Colors: the <<color>> type</h3>
 Images: the <<image>> type</h3>
 
 	The <<image>> data type is defined in [[!CSS3-IMAGES]].
-	UAs that support CSS Image Values Level 3 or its successor must interpret <<image>> as defined therein.
+	UAs that support CSS Image Values Level 3 or its successor
+	must interpret <<image>> as defined therein.
+	UAs that do not yet support CSS Images Level 3
+	must interpret <<image>> as <<url>>.
 
 <!--
 ████████   ███████   ██████  ████ ████████ ████  ███████  ██    ██
diff --git a/css-values/Overview.bs b/css-values/Overview.bs
index beef68a..6b94eac 100644
--- a/css-values/Overview.bs
+++ b/css-values/Overview.bs
@@ -1464,7 +1464,10 @@ Colors: the <<color>> type</h3>
 Images: the <<image>> type</h3>
 
 	The <<image>> data type is defined in [[!CSS3-IMAGES]].
-	UAs that support CSS Image Values Level 3 or its successor must interpret <<image>> as defined therein.
+	UAs that support CSS Images Level 3 or its successor
+	must interpret <<image>> as defined therein.
+	UAs that do not yet support CSS Images Level 3
+	must interpret <<image>> as <<url>>.
 
 <!--
 ████████   ███████   ██████  ████ ████████ ████  ███████  ██    ██
